adapters/IronSource/IronSourceAdapter/GADMAdapterIronSource.m

```diff
@@ -106,21 +109,50 @@ - (void)presentInterstitialFromRootViewController:(UIViewController *)rootViewCo
 #pragma mark Admob Banner
 
 - (void)getBannerWithSize:(GADAdSize)adSize {
-  [self showBannersNotSupportedError];
+    id<GADMAdNetworkConnector> strongConnector = _connector;
+    NSDictionary *credentials = [strongConnector credentials];
+    /* Parse enabling testing mode key for log */
+    self.isLogEnabled = strongConnector.testMode;
+    /* Parse application key */
+    NSString *applicationKey = @"";
+    if (credentials[GADMAdapterIronSourceAppKey]) {
+      applicationKey = credentials[GADMAdapterIronSourceAppKey];
+    }
+
+    /* Parse instance id key */
+    if (credentials[GADMAdapterIronSourceInstanceId]) {
+      self.instanceID = credentials[GADMAdapterIronSourceInstanceId];
+    }
+
+    if ([GADMAdapterIronSourceUtils isEmpty:applicationKey]) {
+      NSError *error = GADMAdapterIronSourceErrorWithCodeAndDescription(
+          GADMAdapterIronSourceErrorInvalidServerParameters,
+          @"'appKey' parameter is missing. Make sure that 'appKey' server parameter is added.");
+      [strongConnector adapter:self didFailAd:error];
+      return;
+    }
+    
+    dispatch_async(dispatch_get_main_queue(), ^{
+        UIViewController *viewController = [_connector viewControllerForPresentingModalView];
+        if (!viewController) {
+                NSError *error =  @"IronSource interstitial ad did load for instance";
```

Comment:
Please use the `GADMAdapterIronSourceErrorWithCodeAndDescription()` utility method for errors.

Reply:
Fixed thanks